install_prerequisites.py

```diff
@@ -104,23 +104,36 @@ def main():
             cmd = ['sudo','apt-get','install','-y','patchelf']
             build.run_cmd(cmd, check_rc='installing patchelf failed')
 
-    elif pld in ['CentOS', 'CentOS Linux', 'Red Hat Enterprise Linux Server', 'Scientific Linux']:
+    elif pld in ['AlmaLinux', 'CentOS', 'CentOS Linux', 'Red Hat Enterprise Linux Server', 'Scientific Linux']:
         log.info('Detected: {0}'.format(pld))
         # prep
-        cmd = ['sudo', 'rpm', '--rebuilddb']
-        build.run_cmd(cmd, check_rc='rpm rebuild failed')
+        if pld == 'AlmaLinux':
+            cmd = ['sudo', 'dnf', 'install', '-y', 'epel-release', 'dnf-plugins-core']
+            build.run_cmd(cmd, check_rc='rpm dnf install failed')
+            cmd = ['sudo', 'dnf', 'config-manager', '--set-enabled', 'powertools']
+            build.run_cmd(cmd, check_rc='rpm dnf config-manager failed')
+            cmd = ['sudo', 'yum', 'install', '-y', 'procps', 'redhat-lsb-core'] # For ps and lsb_release
+            build.run_cmd(cmd, check_rc='yum install failed')
+        else:
+            cmd = ['sudo', 'rpm', '--rebuilddb']
+            build.run_cmd(cmd, check_rc='rpm rebuild failed')
         cmd = ['sudo','yum','clean','all']
         build.run_cmd(cmd, check_rc='yum clean failed')
-        cmd = ['sudo','yum','install','centos-release-scl-rh', '-y']
-        build.run_cmd(cmd, check_rc='yum install failed')
+        #cmd = ['sudo','yum','install','centos-release-scl-rh', '-y']
+        #build.run_cmd(cmd, check_rc='yum install failed')
```

Comment:
There does not appear to be an equivalent package for `centos-release-scl-rh`. I tried to see if CentOS 8 and Rocky Linux provided this package, but they don't appear to do so either.

If anyone has knowledge about this, that would be helpful. I'm planning on reaching out to AlmaLinux people to verify.

Reply:
this was necessary for centos7 - it is not provided for rocky/alma.   i don't remember what it provided that we needed, but it shouldn't just go away, it should just be skipped if not centos7.

Reply:
You are correct. I likely forgot to go back and wrap it in a if-statement.

Reply:
done.